rules.d/99-systemd.rules.in

```diff
@@ -27,7 +27,9 @@ SUBSYSTEM=="block", ENV{DM_UUID}=="CRYPT-*", ENV{ID_PART_TABLE_TYPE}=="", ENV{ID
 # Explicitly set SYSTEMD_READY=1 for DM devices that don't have it set yet, so that we always have something to import above
 SUBSYSTEM=="block", ENV{DM_UUID}=="?*", ENV{SYSTEMD_READY}=="", ENV{SYSTEMD_READY}="1"
 
-# add symlink to GPT root disk
+# Add a stable symlink for the disk that we're booting from
+SUBSYSTEM=="block", ENV{ID_BOOT_DISK}=="1", SYMLINK+="boot-disk"
```

Comment:
i think we should add `ENV{DEVTYPE}=="disk"` here, as safety check. We currently don't set the property on per-partition devices, but we might eventually import the property from the whole block device onto the partition devices (or we might not, but somebody else might in a udev rule), and that shouldn't fuck up our symlink here.

I think this symlink should probably be called /dev/disk/boot, i.e. be placed where the other /dev/disk/by-*/ symlinks are, but as a singleton right inside /dev/disk/

Reply:
actually, this one symlink is so generic, it should really be in rules.d/60-persistent-storage.rules.in I think.